packages/api/source/graphql/privateAPI.ts

```diff
@@ -312,6 +315,25 @@ const typeDefs = gql`
         error: GraphQLJSON,
     }
 
+    enum MembershipTypes{
+        ${membershipTypes.join("\n")}
+    }
+
+    enum MembershipStatuses{
+        ${membershipStatuses.join("\n")}
+    }
+
+    type Membership {
+        id: String!
+        member: String!
```

Comment:
You need to use proper types! Member is of type `User`, not `String`. Make sure all the other types in this schema type follow this.